osu.Game.Tests/Visual/Gameplay/TestSceneSkinEditor.cs

```diff
@@ -40,6 +45,36 @@ public void TestToggleEditor()
             AddToggleStep("toggle editor visibility", visible => skinEditor.ToggleVisibility());
         }
 
+        [Test]
+        public void TestEditComponent()
+        {
+            BarHitErrorMeter hitErrorMeter = null;
+
+            AddStep("select bar hit error blueprint", () =>
+            {
+                var blueprint = skinEditor.ChildrenOfType<SkinBlueprint>().First(b => b.Item is BarHitErrorMeter);
+
+                hitErrorMeter = (BarHitErrorMeter)blueprint.Item;
+                skinEditor.SelectedComponents.Clear();
+                skinEditor.SelectedComponents.Add(blueprint.Item);
+            });
+
+            AddAssert("value is default", () => hitErrorMeter.JudgementLineThickness.IsDefault);
+
+            AddStep("hover first slider", () =>
+            {
+                InputManager.MoveMouseTo(
+                    skinEditor.ChildrenOfType<SkinSettingsToolbox>().First()
+                              .ChildrenOfType<SettingsSlider<float>>().First()
+                              .ChildrenOfType<SliderBar<float>>().First()
+                );
+            });
+
+            AddStep("adjust slider via keyboard", () => InputManager.Key(Key.Left));
+
+            AddAssert("value is less than default", () => hitErrorMeter.JudgementLineThickness.Value < hitErrorMeter.JudgementLineThickness.Default);
+        }
```

Comment:
Is this test no longer required in favour of `TestSceneScreenNavigation.TestEditComponentDuringGameplay`? I would think so given that `SkinEditor` is used here instead of `SkinEditorOverlay` which actually blocks input.

Reply:
I felt like having it isn't an issue, and could be built on going forward.